lexer.d.ts

```diff
@@ -0,0 +1,9 @@
+interface Exports {
+  exports: string[];
+  reexports: string[];
+}
+
+export declare function parse(source: string, name?: string): Exports;
+export declare function init(): Promise<void>;
+
+export {};
```

Comment:
Out of interest, what is this line for?

Reply:
Not 100% sure 😅 I didn't write these definitions manually, I used the TS playground to "implement" this module and copied out the generated `d.ts` file.

https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBAMwK4DsDGMCWEVzAQygGdgAKAWACg44iIko1gAuWmKTFAcwBoqaU+ALbA4AXjgByAAKS+lAJSsA3nFCRYRVkXacuAbQC6AbjhRg66DC1sO3I3AC+cZfzPAYDXKsubWRnndfa39DJ2MqRyoqYMRUDGxcTkwYUiU4AAUoCCFMEgAeADcITAATAD4XN3NPKFwsnLzgADpzOgAbQrIFCMooyiA

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgKIA8AOB7KYDOyA3gLABQyyEWuBAXMvmFKAOYDaAugNzmVQRqOPPgZMWIDj3IBfcuSG1kMAK4gEYYNhDJMcKPggAKfNhVQkY5mwA0yEHAC2KALzIA5AAF3ASgYZhAmI+ZAEwcx0iKhoRBi47AUVY5C5kGV4yOTIFGLBlNQ0tHVBgMCM-ZAAFKGxHYEMAHgA3bGAAEwA+YIpQiHCoHWra+ogAOgFTABsm4x8MrKA

Pick `.D.TS` in the right column. The one with the interface (which I went for) as the empty `export {}` at the end